scripts/linters/general_purpose_linter.py

```diff
@@ -623,22 +620,15 @@ def check_file_type_specific_bad_pattern(filepath, content):
 class GeneralPurposeLinter(python_utils.OBJECT):
     """Manages all the common linting functions. As an abstract base class, this
     is not intended to be used directly.
-
-    Attributes:
```

Comment:
Why removing these?

Reply:
These are duplicates we have these in `__init__` method.